openedx/core/djangoapps/user_api/views.py

```diff
@@ -130,6 +136,25 @@ def post(self, request):
         email = data.get('email')
         username = data.get('username')
 
+        # Begin reCAPTCHA validation #
```

Comment:
After moving this validation in the form, let's please remove these comments.

Reply:
Done removed